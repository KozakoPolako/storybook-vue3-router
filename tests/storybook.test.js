import path from 'path'
import initStoryshots from '@storybook/addon-storyshots';

initStoryshots({
  framework: 'vue3',
  configPath: path.join(__dirname, '../.storybook')
});
