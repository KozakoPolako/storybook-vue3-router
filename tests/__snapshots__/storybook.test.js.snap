// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Basics/ActionBar Many Items 1`] = `<!---->`;

exports[`Storyshots Basics/ActionBar Single Item 1`] = `
<div
  class="my-wrapper"
>
  <div>
    <button
      style="margin-bottom: 1em;"
    >
      Push
    </button>
    <br />
    <button
      style="margin-bottom: 1em;"
    >
      Replace
    </button>
    <br />
    <button
      style="margin-bottom: 1em;"
    >
      Go
    </button>
    <br />
    <button
      style="margin-bottom: 1em;"
    >
      Back
    </button>
    <br />
    <button>
      Forward
    </button>
  </div>
</div>
`;

exports[`Storyshots Basics/Brand/StorybookLogo Normal 1`] = `<!---->`;

exports[`Storyshots Basics/IconButton Active 1`] = `<!---->`;

exports[`Storyshots Basics/IconButton Disabled 1`] = `<!---->`;

exports[`Storyshots Basics/IconButton Icon Button 1`] = `<!---->`;

exports[`Storyshots Basics/IconButton Loading 1`] = `<!---->`;

exports[`Storyshots Basics/IconButton With Text 1`] = `<!---->`;

exports[`Storyshots Basics/IconButton With Text Active 1`] = `<!---->`;

exports[`Storyshots Basics/IconButton With Text Disabled 1`] = `<!---->`;

exports[`Storyshots Basics/Loader Infinite State 1`] = `<!---->`;

exports[`Storyshots Basics/Loader Progress Bar 1`] = `<!---->`;

exports[`Storyshots Basics/Loader Progress Error 1`] = `<!---->`;

exports[`Storyshots Basics/Loader Size Adjusted 1`] = `<!---->`;

exports[`Storyshots Basics/Placeholder Single Child 1`] = `<!---->`;

exports[`Storyshots Basics/Placeholder Two Children 1`] = `<!---->`;

exports[`Storyshots Basics/ScrollArea Both 1`] = `<!---->`;

exports[`Storyshots Basics/ScrollArea Horizontal 1`] = `<!---->`;

exports[`Storyshots Basics/ScrollArea Vertical 1`] = `<!---->`;

exports[`Storyshots Basics/ScrollArea With Outer Border 1`] = `<!---->`;

exports[`Storyshots Basics/Zoom Element Actual Size 1`] = `<!---->`;

exports[`Storyshots Basics/Zoom Element Zoomed In 1`] = `<!---->`;

exports[`Storyshots Basics/Zoom Element Zoomed Out 1`] = `<!---->`;

exports[`Storyshots Docs/ArgRow Array Of 1`] = `<!---->`;

exports[`Storyshots Docs/ArgRow Boolean 1`] = `<!---->`;

exports[`Storyshots Docs/ArgRow Check 1`] = `<!---->`;

exports[`Storyshots Docs/ArgRow Color 1`] = `<!---->`;

exports[`Storyshots Docs/ArgRow Complex Object 1`] = `<!---->`;

exports[`Storyshots Docs/ArgRow Complex Union 1`] = `<!---->`;

exports[`Storyshots Docs/ArgRow Date 1`] = `<!---->`;

exports[`Storyshots Docs/ArgRow Enum 1`] = `<!---->`;

exports[`Storyshots Docs/ArgRow Func 1`] = `<!---->`;

exports[`Storyshots Docs/ArgRow Inline Check 1`] = `<!---->`;

exports[`Storyshots Docs/ArgRow Inline Radio 1`] = `<!---->`;

exports[`Storyshots Docs/ArgRow Long Desc 1`] = `<!---->`;

exports[`Storyshots Docs/ArgRow Long Enum 1`] = `<!---->`;

exports[`Storyshots Docs/ArgRow Long Name 1`] = `<!---->`;

exports[`Storyshots Docs/ArgRow Markdown 1`] = `<!---->`;

exports[`Storyshots Docs/ArgRow Multi Select 1`] = `<!---->`;

exports[`Storyshots Docs/ArgRow Number 1`] = `<!---->`;

exports[`Storyshots Docs/ArgRow Object Of 1`] = `<!---->`;

exports[`Storyshots Docs/ArgRow Radio 1`] = `<!---->`;

exports[`Storyshots Docs/ArgRow Range 1`] = `<!---->`;

exports[`Storyshots Docs/ArgRow Select 1`] = `<!---->`;

exports[`Storyshots Docs/ArgRow String 1`] = `<!---->`;

exports[`Storyshots Docs/ArgRow String Compact 1`] = `<!---->`;

exports[`Storyshots Docs/ArgRow String No Controls 1`] = `<!---->`;

exports[`Storyshots Docs/ArgRow String No Controls Compact 1`] = `<!---->`;

exports[`Storyshots Docs/ArgsTable All Controls 1`] = `<!---->`;

exports[`Storyshots Docs/ArgsTable Compact 1`] = `<!---->`;

exports[`Storyshots Docs/ArgsTable Empty 1`] = `<!---->`;

exports[`Storyshots Docs/ArgsTable Error 1`] = `<!---->`;

exports[`Storyshots Docs/ArgsTable In Addon Panel 1`] = `<!---->`;

exports[`Storyshots Docs/ArgsTable In Addon Panel With Warning 1`] = `<!---->`;

exports[`Storyshots Docs/ArgsTable Loading 1`] = `<!---->`;

exports[`Storyshots Docs/ArgsTable Normal 1`] = `<!---->`;

exports[`Storyshots Docs/ArgsTable Sections 1`] = `<!---->`;

exports[`Storyshots Docs/ArgsTable Sections And Subsections 1`] = `<!---->`;

exports[`Storyshots Docs/ArgsTable Sections Compact 1`] = `<!---->`;

exports[`Storyshots Docs/ArgsTable Subsections Only 1`] = `<!---->`;

exports[`Storyshots Docs/ArgsTable With Default Expanded Args 1`] = `<!---->`;

exports[`Storyshots Docs/ColorPalette Default Style 1`] = `<!---->`;

exports[`Storyshots Docs/ColorPalette Named Colors 1`] = `<!---->`;

exports[`Storyshots Docs/Description Markdown 1`] = `<!---->`;

exports[`Storyshots Docs/Description Markdown Code Snippets 1`] = `<!---->`;

exports[`Storyshots Docs/Description Markdown Links 1`] = `<!---->`;

exports[`Storyshots Docs/Description Text 1`] = `<!---->`;

exports[`Storyshots Docs/DocsPage Empty 1`] = `<!---->`;

exports[`Storyshots Docs/DocsPage Loading 1`] = `<!---->`;

exports[`Storyshots Docs/DocsPage Markdown 1`] = `<!---->`;

exports[`Storyshots Docs/DocsPage No Text 1`] = `<!---->`;

exports[`Storyshots Docs/DocsPage Text 1`] = `<!---->`;

exports[`Storyshots Docs/DocsPage With Subtitle 1`] = `<!---->`;

exports[`Storyshots Docs/EmptyBlock Error 1`] = `<!---->`;

exports[`Storyshots Docs/IconGallery Default Style 1`] = `<!---->`;

exports[`Storyshots Docs/Preview Code Collapsed 1`] = `<!---->`;

exports[`Storyshots Docs/Preview Code Error 1`] = `<!---->`;

exports[`Storyshots Docs/Preview Code Expanded 1`] = `<!---->`;

exports[`Storyshots Docs/Preview Column 1`] = `<!---->`;

exports[`Storyshots Docs/Preview Grid With 3 Columns 1`] = `<!---->`;

exports[`Storyshots Docs/Preview Loading 1`] = `<!---->`;

exports[`Storyshots Docs/Preview Row 1`] = `<!---->`;

exports[`Storyshots Docs/Preview Single 1`] = `<!---->`;

exports[`Storyshots Docs/Preview Wide 1`] = `<!---->`;

exports[`Storyshots Docs/Preview With Additional Actions 1`] = `<!---->`;

exports[`Storyshots Docs/Preview With Centered Multi 1`] = `<!---->`;

exports[`Storyshots Docs/Preview With Centered Single 1`] = `<!---->`;

exports[`Storyshots Docs/Preview With Fullscreen Multi 1`] = `<!---->`;

exports[`Storyshots Docs/Preview With Fullscreen Single 1`] = `<!---->`;

exports[`Storyshots Docs/Preview With Toolbar 1`] = `<!---->`;

exports[`Storyshots Docs/Preview With Toolbar Multi 1`] = `<!---->`;

exports[`Storyshots Docs/SectionRow Collapsed 1`] = `<!---->`;

exports[`Storyshots Docs/SectionRow Nested 1`] = `<!---->`;

exports[`Storyshots Docs/SectionRow Section 1`] = `<!---->`;

exports[`Storyshots Docs/SectionRow Subsection 1`] = `<!---->`;

exports[`Storyshots Docs/Source CSS 1`] = `<!---->`;

exports[`Storyshots Docs/Source JSX 1`] = `<!---->`;

exports[`Storyshots Docs/Source Loading 1`] = `<!---->`;

exports[`Storyshots Docs/Source No Story 1`] = `<!---->`;

exports[`Storyshots Docs/Source Source Unavailable 1`] = `<!---->`;

exports[`Storyshots Docs/Story Error 1`] = `<!---->`;

exports[`Storyshots Docs/Story Inline 1`] = `<!---->`;

exports[`Storyshots Docs/Story Loading 1`] = `<!---->`;

exports[`Storyshots Docs/Story React Hook 1`] = `<!---->`;

exports[`Storyshots Docs/TabbedArgsTable Empty 1`] = `<!---->`;

exports[`Storyshots Docs/TabbedArgsTable Tabs 1`] = `<!---->`;

exports[`Storyshots Docs/TabbedArgsTable Tabs In Addon Panel 1`] = `<!---->`;

exports[`Storyshots Docs/Typeset With Font Family 1`] = `<!---->`;

exports[`Storyshots Docs/Typeset With Font Sizes 1`] = `<!---->`;

exports[`Storyshots Docs/Typeset With Font Weight 1`] = `<!---->`;

exports[`Storyshots Docs/Typeset With Weight Text 1`] = `<!---->`;

exports[`Storyshots Mock Router Default 1`] = `
<div
  class="my-wrapper"
>
  <div>
    <h2>
      $route:
    </h2>
    <pre>
      &lt;pre&gt;{{ §route }}&lt;/pre&gt;
    </pre>
    <pre>
      {
  "isMocked": true,
  "path": "/",
  "fullPath": "/#/",
  "name": "home",
  "meta": {
    "some_meta": "true"
  },
  "params": {
    "some_param": "false"
  },
  "query": {
    "some_query": "false"
  }
}
    </pre>
  </div>
</div>
`;

exports[`Storyshots Mock Router Dynamic Template 1`] = `
<div
  class="my-wrapper"
>
  <div>
    <h2>
      $route:
    </h2>
    <pre>
      &lt;pre&gt;{{ §route }}&lt;/pre&gt;
    </pre>
    <pre>
      {
  "isMocked": true,
  "path": "/",
  "fullPath": "/#/",
  "name": "home",
  "meta": {
    "some_meta": "true"
  },
  "params": {
    "some_param": "false"
  },
  "query": {
    "some_query": "false"
  }
}
    </pre>
  </div>
</div>
`;

exports[`Storyshots Mock Router Programatic Navigation 1`] = `
<div
  class="my-wrapper"
>
  <div>
    <h3
      style="margin-top: 2em;"
    >
      Example dynamic template from 
      <code>
        $route.meta.some_meta
      </code>
    </h3>
    
    <code>
      some_meta
    </code>
     is true! 
    
  </div>
</div>
`;

exports[`Storyshots UI/Addon Panel All Addons 1`] = `<!---->`;

exports[`Storyshots UI/Addon Panel Filtered Addons 1`] = `<!---->`;

exports[`Storyshots UI/Iframe Missing Story 1`] = `<!---->`;

exports[`Storyshots UI/Iframe Preparing Docs 1`] = `<!---->`;

exports[`Storyshots UI/Iframe Preparing Story 1`] = `<!---->`;

exports[`Storyshots UI/Iframe Working Story 1`] = `<!---->`;

exports[`Storyshots UI/Notifications/NotificationItem Accessibility Gold Icon 1`] = `<!---->`;

exports[`Storyshots UI/Notifications/NotificationItem Accessibility Gold Icon Long Head Line No Sub Headline 1`] = `<!---->`;

exports[`Storyshots UI/Notifications/NotificationItem Accessibility Icon 1`] = `<!---->`;

exports[`Storyshots UI/Notifications/NotificationItem Book Icon 1`] = `<!---->`;

exports[`Storyshots UI/Notifications/NotificationItem Book Icon Long Sub Headline 1`] = `<!---->`;

exports[`Storyshots UI/Notifications/NotificationItem Book Icon Sub Headline 1`] = `<!---->`;

exports[`Storyshots UI/Notifications/NotificationItem Link 1`] = `<!---->`;

exports[`Storyshots UI/Notifications/NotificationItem Link Icon With Color 1`] = `<!---->`;

exports[`Storyshots UI/Notifications/NotificationItem Link Icon With Color Sub Headline 1`] = `<!---->`;

exports[`Storyshots UI/Notifications/NotificationItem Long Headline 1`] = `<!---->`;

exports[`Storyshots UI/Notifications/NotificationItem Simple 1`] = `<!---->`;

exports[`Storyshots UI/Notifications/NotificationItem Strong Emphasized Sub Headline 1`] = `<!---->`;

exports[`Storyshots UI/Notifications/NotificationItem Strong Sub Headline 1`] = `<!---->`;

exports[`Storyshots UI/Notifications/NotificationList All 1`] = `<!---->`;

exports[`Storyshots UI/Notifications/NotificationList Placement 1`] = `<!---->`;

exports[`Storyshots UI/Notifications/NotificationList Single 1`] = `<!---->`;

exports[`Storyshots UI/Settings/ReleaseNotes Did Hit Max Wait Time 1`] = `<!---->`;

exports[`Storyshots UI/Settings/ReleaseNotes Loading 1`] = `<!---->`;

exports[`Storyshots UI/Settings/SettingsFooter Basic 1`] = `<!---->`;

exports[`Storyshots UI/Sidebar/Explorer Simple 1`] = `<!---->`;

exports[`Storyshots UI/Sidebar/Explorer With Refs 1`] = `<!---->`;

exports[`Storyshots UI/Sidebar/Search Filled In 1`] = `<!---->`;

exports[`Storyshots UI/Sidebar/Search Last Viewed 1`] = `<!---->`;

exports[`Storyshots UI/Sidebar/Search Shortcuts Disabled 1`] = `<!---->`;

exports[`Storyshots UI/Sidebar/Search Simple 1`] = `<!---->`;

exports[`Storyshots UI/Sidebar/SearchResults Last Viewed 1`] = `<!---->`;

exports[`Storyshots UI/Sidebar/SearchResults No Results 1`] = `<!---->`;

exports[`Storyshots UI/Sidebar/SearchResults Searching 1`] = `<!---->`;

exports[`Storyshots UI/Sidebar/TreeNode Expandable 1`] = `<!---->`;

exports[`Storyshots UI/Sidebar/TreeNode Nested 1`] = `<!---->`;

exports[`Storyshots UI/Sidebar/TreeNode Selection 1`] = `<!---->`;

exports[`Storyshots UI/Sidebar/TreeNode Types 1`] = `<!---->`;
